### PR TITLE
(LEARNVM-631) Change pasture db param default from undef to 'none'

### PR DESCRIPTION
**Pull request by kjhenner**
Changing to a real value will make it easier and clearer later when we
want to manage this value in hiera in a later quest.

### CODE REVIEW

File: us_en/quests/roles_and_profiles.md [hunk 1]
```diff
@@ -276,34 +274,45 @@ To get started creating your node definitions, open your `site.pp` manifest:
...
-`/^pasture-app/` regular expression as the title of your node definition and
-include the `role::pasture_app` role class.
+Create a node definition blocks for your application and database nodes.
+
+Remember, a node definition block can take a regular expression as its title.
+You want any node whose host name begins with `pasture-app` to be classified
+with the `role::pasture_app` role, so create a node definition block with the
+title `/^pasture-app/` and `include` the `role::pasture_app` role for matching
+nodes. Similarly, any node whose name matches `pasture-db` should be classified
+with the `role::pasture_app` role, so add a node definition with the title
```
Comment (greglarkin) [inline us_en/quests/roles_and_profiles.md]: Is this the correct role class to include on the db node?

**Comment by kjhenner** [comment]
@greglarkin Good catch. I amended the commit to specify the correct role there.

File: us_en/quests/roles_and_profiles.md [hunk 2]
```diff
@@ -276,34 +274,45 @@ To get started creating your node definitions, open your `site.pp` manifest:
...
 ```puppet
+node default {
+  # This is where you can declare classes for all nodes.
+  # Example:
+  #   class { 'my_class': }
+}
+
 node /^pasture-app/ {
   include role::pasture_app
 }
```
Comment (greglarkin) [inline us_en/quests/roles_and_profiles.md]: Should the node definition for `^pasture-db` also be included here?

Comment (kjhenner) [reply us_en/quests/roles_and_profiles.md]: @greglarkin If you look at the full changes for the file, you'll see that it is. The way GH displays the changes just puts some deleted lines in the midst of the site.pp file code block.

Comment (greglarkin) [reply us_en/quests/roles_and_profiles.md]: Great, thank you @kjhenner

**Pull request merged**